addons/l10n_ar/models/account_journal.py

```diff
@@ -89,13 +89,13 @@ def _get_journal_codes(self):
             return usual_codes + receipt_codes + expo_codes + invoice_m_code + receipt_m_code
         elif self.l10n_ar_afip_pos_system in ['RAW_MAW', 'RLI_RLM']:
             # electronic/online invoice
-            return usual_codes + receipt_codes + invoice_m_code + receipt_m_code + mipyme_codes + liq_product_codes
+            return usual_codes + receipt_codes + invoice_m_code + receipt_m_code + mipyme_codes + liq_product_codes + expo_codes
         elif self.l10n_ar_afip_pos_system in ['CPERCEL', 'CPEWS']:
             # invoice with detail
-            return usual_codes + invoice_m_code
+            return usual_codes + invoice_m_code + expo_codes
         elif self.l10n_ar_afip_pos_system in ['BFERCEL', 'BFEWS']:
             # Bonds invoice
-            return usual_codes + mipyme_codes
+            return usual_codes + mipyme_codes + expo_codes
```

Comment:
@MerlinGuillaume this expo_codes shouldn't be added here as they won't be used in this cases
Then the change will be adding the "E" as an option (line 52).
And that will solve the issue reported by the customer.
cc' @jjscarafia @jco-odoo @fvz-odoo @ren-odoo

Reply:
Alright, thanks !